tests/test_randomproto.py

```diff
@@ -1,20 +1,16 @@
-"""Test randomproto."""
 import pytest
 
 import randomproto
-import example2_pb2
-import example3_pb2
+from tests.proto3.example1_pb2 import TypeMessage
```

Comment:
Are you dropping support for proto2? I'd like to keep both proto2 and proto3 support in this package.

Reply:
Isn't proto2 deprecated? Maybe I'm wrong here

Reply:
From https://protobuf.dev/programming-guides/proto2/, I didn't see proto2 is deprecated.

> It is also unclear how to set up a dev environment to test both proto3 (new + old) and proto2 generated files.

What do you mean by "new + old" proto3?

Reply:
I mean old is generated using the old protoc compiler by you, and new is generated by my 5.26.0 compiler.

Reply:
I've added PROTOCOL_BUFFERS_PYTHON_IMPLEMENTATION to conftest.py to fix the tests run.

Reply:
Thanks. That looks good to me.